server/services/caseloadService.ts

```diff
@@ -95,8 +95,30 @@ export default class CaseloadService {
       })
 
     const combinedCases = await Promise.all([casesWithLicences, casesPendingLicence]).then(c => c.flat())
```

Comment:
filterByPrisonOrProbationView is the key function. It is called here because all the existing logic such as for ensuring results only include releases within 4 weeks, is already executed.
Essentially, we are splitting the single view user previously saw into 2 parts based on the licence status.
The Variation type statuses are however an addition to what the user could previously see.